src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInternalTransaction.cs

```diff
@@ -526,15 +486,19 @@ private void ZombieParent()
         {
             if (_parent != null && _parent.TryGetTarget(out SqlTransaction parent))
             {
-                 parent.Zombie();
+                parent.Zombie();
             }
             _parent = null;
         }
 
-        internal string TraceString()
-        {
-            return string.Format(/*IFormatProvider*/ null, "(ObjId={0}, tranId={1}, state={2}, type={3}, open={4}, disp={5}",
-                        ObjectID, _transactionId, _transactionState, _transactionType, _openResultCount, _disposing);
-        }
+        internal string TraceString() => string.Format(/*IFormatProvider*/ null, 
+            "(ObjId={0}, tranId={1}, state={2}, type={3}, open={4}, disp={5}",
+            ObjectID, _transactionId, _transactionState, _transactionType, _openResultCount, _disposing);
+
+#if NETFRAMEWORK
+
+        internal bool IsContext => TransactionType.Context == _transactionType;
```

Comment:
Does this have callers? If so what functionality do they provide on netfx that isn't in netcore? does netcore go without this method in a way that netfx can do?

Reply:
@Wraith2 IsContext is being used by netfx as a condition for connection deactivation. I have to look into if NetCore is missing this or Netfx can move forward without it. I will follow-up here on this part once I get the clear picture but I would address that as a separate PR if there is possible solution and leave this piece of code here for Code merge PR as of now.

Reply:
Looking into code, I see the change cant be merged or removed. The property is netfx specific and critical condition for connection deactivation for netfx implementation.